### PR TITLE
Remove processing logline as its not really needed

### PR DESCRIPTION
**Pull request by bobbytables**
This doubles logs for all grpc endpoints when its really not useful.

### CODE REVIEW

File: grpc/interceptor/logger.go [hunk 1]
```diff
@@ -51,7 +49,12 @@ func Logger(l *logrus.Logger) grpc.UnaryServerInterceptor {
 	}
 }
 
-func addLoggerToContext(l *logrus.Logger, ctx context.Context) context.Context {
-	entry := l.WithField("request_id", uuid.NewV4().String())
+func addLoggerToContext(ctx context.Context, l *logrus.Logger) context.Context {
+	id, err := uuid.NewV4()
+	if err != nil {
+		logrus.WithError(err).Fatal("could not generate request id")
+	}
```
Comment (bobbytables) [inline grpc/interceptor/logger.go]: This isn't ideal but I honestly have no idea why it would ever fail to generate 🤷‍♂️

**Comment by bobbytables** [comment]
slack_-_namely

File: grpc/interceptor/logger.go (hunk 1 above)
Comment (mhamrah) [reply grpc/interceptor/logger.go]: it relies on time, and the underlying time implementation can fail. i believe google's uuid has a wrapper w/ no error that will panic

Comment (bobbytables) [reply grpc/interceptor/logger.go]: I figured that would be the error but the odds of that are so low that I'm cool with a crash. Somethings aren't recoverable.

**Pull request merged**